lib/mongodb.js

```diff
@@ -483,6 +483,25 @@ MongoDB.prototype.updateOrCreate = function updateOrCreate(model, data, options,
 };
 
 /**
+ * Replace if the model instance exists with the same id or create a new instance
+ *
+ * @param {String} model The model name
+ * @param {Object} data The model instance data
+ * @param {Object} options The options object
+ * @param {Function} [cb] The callback function
+ */
+MongoDB.prototype.replaceOrCreate = function(model, data, options, cb) {
+  if (this.debug) {
```

Comment:
{} not required for single line statements

```
if (this.debug)
  debug('replaceOrCreate', model, data);

var id = ...
```

Same goes for all the other debugs.